include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -1913,37 +1950,52 @@ namespace internal
                      Number *                                      values_dofs,
                      Number *                                      values_quad,
                      Number *           gradients_quad,
+                     Number *           hessians_quad,
                      Number *           scratch_data,
                      const bool         evaluate_val,
                      const bool         evaluate_grad,
+                     const bool         evaluate_hessian,
```

Comment:
This reminds me of the fact that we should convert this to the `EvaluationFlags` argument like for the cell integrators, but we don't need it for this PR.

Reply:
I totally agree. Having those three bools in all interfaces is just ugly, but this is something for a follow up.